page_analyzer/templates/base.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Аналиратор страниц</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body class="min-vh-100 d-flex flex-column">
+    <header class="flex-shrink-0">
+      <nav class="navbar navbar-expand-md navbar-dark bg-dark px-3">
+        <a class="navbar-brand" href="/">Анализатор страниц</a>
+        <div id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link" href="{{ url_for('get_urls') }}">Сайты</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      {% with messages = get_flashed_messages(with_categories=True) %} {% if
```

Comment:
let's put `{% if ...` on a separate lane with indentation, e.g.  

```suggestion
      {% with messages = get_flashed_messages(with_categories=True) %}
        {% if messages %}
```

Reply:
can do nothing about it.
it's the way pre-commit works, not my fault))

Reply:
what happens if you put it on a separate line and add indentation? like in the example above

Reply:
didn't help.
 turned the same strange way

Reply:
ok, let's keep it as it is then - prettier doesn't support jinja templates
another option - drop prettier, but I prefer the first option more